bin/newman

```diff
@@ -21,10 +21,25 @@ var color   = require('cli-color'),
 
 
 
+function parseConfigFile() {
+	var config,
+		option;
+
+	Helpers.validateConfigFile(program.configFile);
+	config = Helpers.validateAndReturnConfig(program.configFile);
+
+	for (option in config) {
+		if (!program[option]) {
```

Comment:
`!program[option]` would pass for `false` and zero. Not sure whether commander parses to Boolean and number, but surely checking for own property or inequality with null and undefined is better.